### PR TITLE
Tweak search

### PR DESCRIPTION
**Pull request by goodyduru**
- Changed structure of table results output to differentiate between results containing all the rows in a columns from those containing all the columns in a row.

- Added ability to search through text chunks.

### CODE REVIEW

File: api/models/extract.py [hunk 1]
```diff
@@ -686,27 +686,46 @@ def extract_date(chunk):
...
+    return title, results
+
+def search_table_chunks(tables, texts):
+    results = []
     for table in tables:
         result = find_text(table, texts)
         if len(result) > 0:
-            text_results.append(result)
-    return title, text_results
+            print(result)
```
Comment (otseobande) [inline api/models/extract.py]: Is this `print` statement need in production?

File: api/models/extract.py [hunk 2]
```diff
@@ -837,4 +891,8 @@ def find_investor_info(path, investor_name, investor_dict):
                         break
         if investor_key not in finds:
             finds[investor_key] = 'N/A'
-    return finds
\ No newline at end of file
+    return finds
+
+if __name__ == "__main__":
+    search = search("../files/doc.pdf", "AIP")
+    print(search)
```
Comment (otseobande) [inline api/models/extract.py]: It would be cool if we wrote a test for the `search` function.

File: api/models/extract.py (hunk 1 above)
Comment (goodyduru) [reply api/models/extract.py]: Wow....totally forgot about that. Thanks for pointing that out.

File: api/models/extract.py (hunk 2 above)
Comment (goodyduru) [reply api/models/extract.py]: I've thought about that tbh, but this script is kinda hacky.

Comment (otseobande) [reply api/models/extract.py]: Can you give it a shot though?

File: api/models/extract.py (hunk 1 above)
Comment (otseobande) [reply api/models/extract.py]: lool

File: api/models/extract.py (hunk 2 above)
Comment (goodyduru) [reply api/models/extract.py]: I will try

**Pull request merged**